relayer-root.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
consider naming this file `LetsEncrypt-X1-Root.pem` to show what it is, or `relayer-mainnet-root.pem` to show what it's for.